packages/pipeline-editor/src/index.ts

```diff
@@ -113,6 +114,20 @@ const extension: JupyterFrontEndPlugin<void> = {
       name: widget => widget.context.path
     });
 
+    // Add command to add file to pipeline
+    const addFileToPipelineCommand: string = commandIDs.addFileToPipeline;
+    app.commands.addCommand(addFileToPipelineCommand, {
+      label: 'Add file to pipeline',
```

Comment:
```suggestion
      label: 'Add File to Pipeline',
```
capitalization should match other menu items